components/collapsible-panel/collapsible-panel.js

```diff
@@ -452,21 +459,23 @@ class CollapsiblePanel extends SkeletonMixin(FocusMixin(RtlMixin(LitElement))) {
 	}
 
 	_renderPanelTitle() {
-		let headingStyle = (this.headingStyle === defaultHeading && this.headingLevel !== this.headingStyle) ? this.headingLevel : this.headingStyle;
-		headingStyle = normalizeHeadingNumber(headingStyle);
+		const headingStyle = (this.headingStyle === defaultHeading && this.headingLevel !== this.headingStyle) ? this.headingLevel : this.headingStyle;
+		this.headingStyle = normalizeHeadingStyle(headingStyle);
 
 		const titleClasses = {
 			'd2l-collapsible-panel-title': true,
 			'd2l-skeletize': true,
-			[`d2l-heading-${headingStyle}`]: true,
+			[`d2l-heading-${this.headingStyle}`]: true,
```

Comment:
I noticed that `heading-style` was appearing in the DOM as the default "3", regardless of what the component did, since we never update `this.headingStyle`.

Figured it made sense to set the property here as well...shouldn't change any functionality...